proposals/drafts/elections.rst

```diff
@@ -1,37 +1,249 @@
+
+VIP 5, the specification's current version, incorporates elements from the `Election Results Common Data Format Specification <https://www.nist.gov/itl/voting/nist-election-results-common-data-format-specification>`_ defined by the National Institute of Standard and Technology. As such, we have borrowed eagerly from NIST's current specification also.
+
+Important differences between this proposal and the current VIP specification are noted below.
+
+Questions
+=========
+
+* Does ``Election`` need one or two properties to store something like the election type (e.g., general, primary, special, runoff) or election level (e.g., federal, state, county, local)?
+* Should ``Party`` be implemented as an ``Organization`` (or subclass)? If so, how do we handle national parties versus state parties (e.g., the DNC versus Missouri Democratic Party)? Would probably be more accurate to associate state and local candidates with state parties and federal candidates with the national parties. But would expect most users typically to want all Democrats to be grouped together regardless of the level of government, especially when analyzing election results.
+* Should the proposed subclasses of OCD data types (e.g., ``Election``, ``BallotMeasureContest``,  ``CandidateContest``) each implement its own ID or should it just inherit the id field of the base class?
+* Do we need to support breaking down either contest summary counts and/or vote counts by other geographies? Voting devices? (Both of these are supported in NIST).
+
 
 Implementation
 ==============
 
 Election
 --------
 
+A collection of political contests set to be decided on the same date.
+
+``Election`` is a subclass of OCD's ``Event`` data type, defined in `OCDEP 4: Events <http://opencivicdata.readthedocs.io/en/latest/proposals/0004.html>`_, which was accepted in June 2014. All of core and optional properties of ``Event`` are inherited by ``Election``.
+
+The typical implementation will be an ``all_day`` event witht a "election" ``classification`` value and a ``start_time`` set to midnight of the observed election date.
+
 id
-    ocd identifier
+    Open Civic Data-style id in the format ``ocd-election/{{uuid}}``.
 
 identifiers
-    upstream identifer
+    **optional**
+    **repeated**
+    Upstream identifiers of the election if any exist, such as those assigned by a Secretary of State, county or city elections office.
+
+administrative_org_id
+    **optional**
+    Reference to the ``Organization`` that administers the election.
+
+state
+    `FIPS code <https://en.wikipedia.org/wiki/Federal_Information_Processing_Standard_state_code>`_ of the state where the election is being held. Recorded in the format ``st{{fips}}`` to match references to VIP `<State> <http://vip-specification.readthedocs.io/en/release/built_rst/xml/elements/state.html>`_  elements (string).
+
+is_statewide
+    **optional**
+    Indicates whether the election is statewide (boolean).
```

Comment:
This seems overly narrow. What about, say, countywide elections? Should there be a pointer to the jurisdiction an election covers instead? Alternatively, while this seems like a useful field, what would it actually be used for? A jurisdiction pointer would allow us to fetch all elections covering a jurisdiction; what would is_statewide be used for?